lib/lure/command/updateDependencies.go

```diff
@@ -38,10 +38,10 @@ func appendIfMissing(modules []versionManager.ModuleVersion, modulesToAdd []vers
 }
 
 func CheckForUpdatesJobCommand(project project.Project, sourceControl sourceControl, repository repository, args map[string]string) error {
-	return checkForUpdatesJob(project, sourceControl, repository, args["commitMessage"])
+	return checkForUpdatesJob(project, sourceControl, repository, args["commitMessage"], args["pullRequestDescription"])
```

Comment:
i think this will break if the config doesn't include a description? you either should add a default value somewhere or fallback on the commit message when there's no description

Reply:
It seems like one is set if none is provided : https://github.com/coveooss/lure/blob/792d5f15be08e67a23e98db10fc66e6567944909/lib/lure/project/project.go#L80